lib/prawn/graphics/color.rb

```diff
@@ -94,7 +94,11 @@ def process_color(*color)
       def color_type(color)
         case color
         when String
-          :RGB
+          if color =~ /\A[0-F]{6}\z/i
```

Comment:
There are a few characters that we probably don't want to match between `'9'` and `'A'`. I'd prefer this regex to be something more specific. For example `/\A[0-9a-f]{6}\z/i`.